atakama/rule_engine.py

```diff
@@ -168,9 +172,16 @@ class RuleSet(List[RulePlugin]):
 
     def approve_request(self, request: ApprovalRequest) -> bool:
         """Return true if all rules return true."""
-        for rule in self:  # pylint: disable=not-an-iterable
+        for i, rule in enumerate(self):  # pylint: disable=not-an-iterable
             try:
                 res = rule.approve_request(request)
+                log.debug(
```

Comment:
we can definitely do the load-time info thing in this file, since it's not ever runtime loaded, to help reduce spam

Reply:
done